modules/apps/collaboration/wiki/wiki-editor-configuration/src/test/java/com/liferay/wiki/editor/configuration/internal/WikiAttachmentImageHTMLEditorConfigContributorTest.java

```diff
@@ -165,6 +170,11 @@ public void testItemSelectorURLWhenAllowBrowseAndValidWikiPage()
 		wikiAttachmentImageHTMLEditorConfigContributor.setItemSelector(
 			_itemSelector);
 
+		suppress(
```

Comment:
Why is this needed??

At first sight this change doesn't seem to be needed...

Why don't you need to suppress other similar methods??

Thanks for the info